File: LV1/2016년.js
function solution(a, b) {
  let month = [31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31];
  let day = ["THU", "FRI", "SAT", "SUN", "MON", "TUE", "WED"];
  let today = 0;

  for (let i = 0; i < a - 1; i++) {
    today += month[i];
  }
  return day[(today + b) % 7];
}

File: LV1/2016년.go
func solution(a int, b int) string {
	day := []string{"FRI", "SAT", "SUN", "MON", "TUE", "WED", "THU"}
	month := []int{31,29,31,30,31,30,31,31,30,31,30,31}
	sum := 0
	for i:=0; i<a-1; i++ {
		sum += month[i]
	}
	sum += b-1
	if sum >= 0 {
		return day[sum%7]
	}
	return ""
}